core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -835,6 +835,11 @@ public static Jenkins getInstance() {
      */
     private Boolean noUsageStatistics;
 
+    /**
+     * True to enable the new UI that came out of the work of the UX SIG
+     */
+    private Boolean useNewUI;
```

Comment:
Is it standard to use Boolean instead of Boolean?

Reply:
At least it's the one used in this class

Reply:
I would make the types consistent, have you tested this with jcasc? I remember an issue when someone was mixing the types

Reply:
I haven't tested. To be honest I supposed it would work since I'm following a similar approach to other fields and I didn't consider that would cause a problem. Right now I'm not sure I have availability in the short term to check, but I will test it for sure

Reply:
tested and it seems to work fine with JCasC